colour/appearance/rlab.py

```diff
@@ -273,6 +273,7 @@ def XYZ_to_RLAB(
     M = np.matmul(np.matmul(MATRIX_R, row_as_diagonal(LMS_a_L)), MATRIX_XYZ_TO_HPE)
     XYZ_ref = vecmul(M, XYZ)
 
+    Y_ref: NDArrayFloat
```

Comment:
This ensures that the right `@overload` of `spow` is selected, so that `LR` will be inferred as `NDArrayFloat`